inc/db_pgsql.php

```diff
@@ -676,13 +681,13 @@ function field_exists($field, $table)
 	/**
 	 * Add a shutdown query.
 	 *
-	 * @param resource The query data.
-	 * @param string An optional name for the query.
+	 * @param resource $query The query data.
+	 * @param string $name An optional name for the query.
 	 */
-	function shutdown_query($query, $name=0)
+	function shutdown_query($query, $name="")
 	{
 		global $shutdown_queries;
-		if($name)
+		if(!empty($name))
```

Comment:
I don't think it was forced for the name to be an string so I wouldn't go with `trim`, `empty` seems fine.